Makefile

```diff
@@ -101,7 +101,10 @@ upgrade-deps:
 	rm requirements/app.txt
 	rm requirements/test.txt
 
+ifneq ($(skip-test), yes)
+# Run tests only if not in docker
```

Comment:
The comment should be updated, too, but then becomes redundant in this place, given the clarity of the if-statement. It might be useful near the call to `make upgrade-deps`, though. For me, that's the place where the rationale for skipping the tests should be explained. "Skip tests while running in docker, because ...")